packages/volto/src/components/manage/Sidebar/Sidebar.jsx

```diff
@@ -51,23 +63,34 @@ const Sidebar = (props) => {
     blockTab,
     settingsTab,
     orderTab = true,
+    FavoriteTab = true,
```

Comment:
```suggestion
    favoriteTab = true,
```
it's not a constant or a component. Please start variable name with lowercase letter